src/goto-programs/show_goto_functions.cpp

```diff
@@ -51,9 +53,19 @@ void show_goto_functions(
 
   case ui_message_handlert::uit::PLAIN:
     {
+      // sort alphabetically
```

Comment:
Why is this only done in the `PLAIN` output? It might make the most sense for that case, but then at least the commit message should be clear about that.

Reply:
Ok, will sort all